workers/unity/Assets/Playground/Scripts/Worker/WorkerConnectorBase.cs

```diff
@@ -54,6 +54,11 @@ public async Task Connect(string workerType, ILogDispatcher logger)
                         new LogEvent("Cancelling connection attempt")
                             .WithField("WorkerType", workerType)
                             .WithField("Reason", "Required worker failed to connect"));
+#if UNITY_EDITOR
+                    logger.HandleLog(LogType.Warning,
```

Comment:
Arguably this is actually a Debug.Log as this is a pure editor message and not a worker message.

Reply:
I wasn't sure, as it is a "warning" in the sense that it is warning the user about a potential issue. Do you think it is never appropriate to use a warning for a notification like that?